cdap-ui/app/features/admin/templates/namespace/dataset-metadata.html

```diff
@@ -63,4 +63,8 @@
     <td>Is Dataset</td>
     <td>{{ metadata.from_dataset }}</td>
   </tr>
-</table>
\ No newline at end of file
+</table>
+
+<div class="well text-center" ng-if="!metadata">
+  <h4>No metadata for this dataset</h4>
```

Comment:
If metadata is null, doesn't that mean that the dataset does not exist? (as opposed to its metadata simply missing)?